src/app/data/forms/checkbox.ts

```diff
@@ -3,6 +3,7 @@ import { FormControl } from './form-control';
 
 @XmlClass('checkbox')
 export class Checkbox extends FormControl {
+  value = '';
```

Comment:
Da alle FormControls ein value haben, wäre es am besten, das in die abstrakte Basisklasse aufzunehmen, anstatt in jedem einzelnen FormControl

Reply:
super, macht sinn. die Struktur hatte sich etwas geändert, ziehe ich nach. 
Als Ergänzung: Button ist jetzt auch von TitleControl abgeleitet, ziehe ich auch nach und verwende title von Basisklasse.